websites/N/Notespace/metadata.json

```diff
@@ -4,15 +4,15 @@
 		"id": "630812692659044352",
 		"name": "itsmateo20"
 	},
-	"service": "Notespace",
+	"service": "notespace",
 	"description": {
-		"en": "Notespace is your learning platform. Here you will find tons of notes, videos and much more, all for free!",
-		"pl": "Notespace to twoja plaforma do nauki. Znajdziesz tutaj mnóstwo notatek, filmów i wiele więcej, a to wszystko za darmo!"
+		"en": "notespace is your learning platform. Here you will find tons of notes, videos and much more, all for free!",
+		"pl": "notespace to twoja plaforma do nauki. Znajdziesz tutaj mnóstwo notatek, filmów i wiele więcej, a to wszystko za darmo!"
 	},
 	"url": "notespace.edu.pl",
-	"version": "1.0.2",
-	"logo": "https://cdn.rcd.gg/PreMiD/websites/N/Notespace/assets/logo.png",
-	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/N/Notespace/assets/thumbnail.png",
+	"version": "1.0.3",
+	"logo": "https://cdn.rcd.gg/PreMiD/websites/N/notespace/assets/logo.png",
+	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/N/notespace/assets/thumbnail.png",
```

Comment:
Undo.
```suggestion
	"logo": "https://cdn.rcd.gg/PreMiD/websites/N/Notespace/assets/logo.png",
	"thumbnail": "https://cdn.rcd.gg/PreMiD/websites/N/Notespace/assets/thumbnail.png",
```

Reply:
Yeah, if change the uppercase letter N to lowercase, the cdn link will break: 
![image](https://github.com/PreMiD/Presences/assets/68751594/f4e3317c-36c8-4250-8f12-5afd3e644028)

Reply:
Make them imgur links again, or another cdn, and it will reupload with the correct caps.